Scripts/debug/debug_blast_parser.py

```diff
@@ -41,7 +42,9 @@
 
 -o    <testfile> is a BLAST output file.
 
-""" % sys.argv[0]
+"""
```

Comment:
This closing triple quote doesn't align with the opening one. not sure if we want them aligned as for instance the parentheses do not align either, black style does that; I have mostly aligned DocStrings but assignments I might have left them like this.

Reply:
I haven't explored the rules of the syntax to properly understand how Python interprets the indentation vs which leading spaces are part of the string values.  However, this does look like a style judgement - in which case do whatever black does is the easy solution.  This is worth discussing for #2008 as a whole.